src/components/Letters/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import styled, { css } from "styled-components";
+import chroma from "chroma-js";
+
+import "./styles.css";
+
+export const Item = styled.div`
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@500&display=swap');  
+display: flex;
+  justify-content: center;
+  font-family: 'Quicksand';
+  padding: .2rem;
+  ${({ color = chroma.random() }) =>
```

Comment:
I think the random colors looks great, but let's run it by the UI/UX team first.